Rent.Service/Rent.Service.Infrastructure/Constants/DatabaseConstants.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Rent.Service.Infrastructure.Consts;
+
+public class DatabaseConstants
+{
+    public const string DbConnection = "DefaultConnection";
```

Comment:
Redundant approache just to create one constant for `DefaultConnection` even if you hate inline strings in code